libsrc/transpose.code

```diff
@@ -8,12 +8,12 @@
 
 ! pjj/cray
   if(IO == 'write') then
-!dir$ concurrent
+!!dir$ concurrent
```

Comment:
What are these changes all about? Mistake?

Reply:
It's some compiler directive for Cray. Adding the extra `!` disables the it, like OpenMP.